hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieRecord.java

```diff
@@ -31,6 +33,13 @@
   public static String PARTITION_PATH_METADATA_FIELD = "_hoodie_partition_path";
   public static String FILENAME_METADATA_FIELD = "_hoodie_file_name";
 
+  public static List<String> HOODIE_META_COLUMNS = new ImmutableList.Builder<String>().add(COMMIT_TIME_METADATA_FIELD)
```

Comment:
The meta columns is present in HoodieAvroUtils. Can you access this from there ?

Reply:
The meta-columns are defined in HoodieRecord and used in HoodieAvroUtils. Did not find an existing collection in HoodieAvroUtils to use. BTW, I will also mark this field final.